src/components/VotesBar.jsx

```diff
@@ -1,10 +1,39 @@
 import React from "react";
+import { useState } from "react";
+import { useEffect } from "react";
+import { patchArticleVotes } from "../api";
+
+const VotesBar = ({ article }) => {
+  const [votes, setVotes] = useState();
+
+  const handleClick = (newVote) => {
+    setVotes(votes + newVote);
+    patchArticleVotes(article.article_id, newVote).then(() => {});
```

Comment:
What happens if this fails? The user should be able to see that their vote was unsuccessful